### PR TITLE
Release/1.1.0

### PR DESCRIPTION
**Pull request by jakobbraun**

### CODE REVIEW

File: doc/changes/changes_1.1.0.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 Code name: Parquet support
 
+## Summary
```
Comment (AnastasiiaSergienko) [inline doc/changes/changes_1.1.0.md:5]: Maybe you can change the release date right away so you don't need to make additional commit later

File: src/main/java/com/exasol/adapter/document/files/FilteringIterator.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
+package com.exasol.adapter.document.files;
+
+import java.util.Iterator;
+import java.util.NoSuchElementException;
+import java.util.function.Predicate;
+
+/**
+ * This iterators wraps another iterator an filters it's result based on a predicate.
```
Comment (AnastasiiaSergienko) [inline src/main/java/com/exasol/adapter/document/files/FilteringIterator.java:8]:
```suggestion
 * This iterators wraps another iterator and filters its result based on a predicate.
```

File: src/main/java/com/exasol/adapter/document/files/S3FileLoader.java [hunk 3]
```diff
@@ -57,12 +58,14 @@ private AwsCredentials getCredentials(final ExaConnectionInformation exaConnecti
...
 
     @Override
-    public Source<LoadedFile, NotUsed> loadFiles() {
+    public Iterator<LoadedFile> loadFiles() {
         final com.exasol.adapter.document.files.stringfilter.matcher.Matcher filePatternMatcher = this.filePattern
                 .getDirectoryIgnoringMatcher();
-        return getObjectKeysOnlyQuickFiltered()
-                .filter(s3Object -> filePatternMatcher.matches(s3Object.getUri().toString()))
-                .filter(object -> this.segmentMatcher.matches(object.getUri().getKey())).map(this::getS3Object);
+        final Iterator<S3ObjectDescription> objectKeys = getObjectKeysOnlyQuickFiltered();
```
Comment (AnastasiiaSergienko) [inline src/main/java/com/exasol/adapter/document/files/S3FileLoader.java:64]:
```suggestion
        final Iterator<S3ObjectDescription> objectKeys = getQuickFilteredObjectKeys();
```

File: src/test/java/com/exasol/adapter/document/files/FilteringIteratorTest.java [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+import java.util.ArrayList;
+import java.util.List;
+
+import org.hamcrest.Matchers;
+import org.junit.jupiter.api.Test;
+
+class FilteringIteratorTest {
+
+    @Test
+    void test() {
```
Comment (AnastasiiaSergienko) [inline src/test/java/com/exasol/adapter/document/files/FilteringIteratorTest.java:14]: test?

**Review by morazow — APPROVED** [review]

**Pull request merged**